autogtp/README.md

```diff
@@ -4,21 +4,45 @@ This is a self-play tool for Leela-Zero. When launched, it will fetch the
 best network from the server so far, play a game against itself, and upload
 the SGF and training data at the end of the game.
 
+To work successfully, autogtp binary needs to also have leelaz binary to be in the same directory.
+
 # Requirements
 
 * Qt 5.3 or later with qmake
 * C++14 capable compiler
 * curl
 * gzip and gunzip
 
-## Example of compiling - Ubuntu
+## Compiling - Ubuntu and similar
```

Comment:
These instructions are all pointless IMHO. The only reason when you're going to deal with compiling AutoGTP separately is if you're going to work on it, and I think in that case detailed instructions on how to install Qt are not helpful. Unlike the main README we can assume people know what they are doing here.

Reply:
ok, this is why i removed them in the first place, i also think that they are not needed at all for typical use

but you said earlier that it didnt make sense for the autogtp readme, thats why i reintegrated them

i'll remove them again then